client/transaction-pool/src/graph/pool.rs

```diff
@@ -1077,6 +1076,32 @@ mod tests {
 			assert_eq!(stream.next(), Some(TransactionStatus::Dropped));
 		}
 
+		#[test]
+		fn should_trigger_dropped_lower_priority() {
+			// given
+			let limit = Limit { count: 1, total_bytes: 1000 };
+			let options = Options { ready: limit, future: limit, ..Default::default() };
+
+			let pool = Pool::new(options, true.into(), TestApi::default().into());
+
+			let xt = Extrinsic::IncludeData(Vec::new());
+			block_on(pool.submit_one(&BlockId::Number(0), SOURCE, xt)).unwrap();
+			assert_eq!(pool.validated_pool().status().ready, 1);
+
+			// then
+			let xt = uxt(Transfer {
+				from: AccountId::from_h256(H256::from_low_u64_be(2)),
+				to: AccountId::from_h256(H256::from_low_u64_be(1)),
+				amount: 4,
+				nonce: 1,
+			});
+			let result = block_on(pool.submit_one(&BlockId::Number(1), SOURCE, xt));
```

Comment:
Could you also show that adding a tx with a higher priority drops the other?

Reply:
Added